ocs_ci/deployment/deployment.py

```diff
@@ -784,33 +784,41 @@ def setup_local_storage():
 
 
 @retry(AssertionError, 12, 10, 1)
-def verify_pvs_created(num_workers):
+def verify_pvs_created(num_of_pvs):
```

Comment:
`num_of_pvs` is too similar to `num_pvs` which we use to verify below. Can we rename this to something like `expected_pvs`?

Reply:
Done